csv_lua/sym_code.py

```diff
@@ -0,0 +1,18 @@
+class sym:
+  def __init__(self) -> None:
+    self.n = 0
+    self.at = 0
+    self.name = ""
+    self._has = dict()
+
+  def add(self,v) -> None:
+    if v!='?':
+      self.n = self.n+1
+      if v not in self._has:
```

Comment:
If you want to avoid the if/else thingy, you can do : self._has[v] = self._has.get(v,0) + 1. 
Just a suggestion.

Reply:
Resolved